src/components/swapv2/LimitOrder/ListLimitOrder/TabSelector.tsx

```diff
@@ -0,0 +1,46 @@
+import { t } from '@lingui/macro'
+import { useMemo } from 'react'
+import { useMedia } from 'react-use'
+import { Flex } from 'rebass'
+
+import TabButton from 'components/TabButton'
+import { MEDIA_WIDTHS } from 'theme'
+
+import { LimitOrderTab } from '../type'
+
+export default function TabSelector({
+  activeTab,
+  setActiveTab,
+}: {
+  activeTab: LimitOrderTab
+  setActiveTab: (n: LimitOrderTab) => void
+}) {
+  const upToSmall = useMedia(`(max-width: ${MEDIA_WIDTHS.upToSmall}px)`)
+  const style = useMemo(
+    () => ({ padding: '16px', flex: 'unset', fontSize: '14px', width: upToSmall ? '50%' : 'fit-content' }),
+    [upToSmall],
+  )
+
+  return (
+    <Flex
+      sx={{
+        borderTopLeftRadius: upToSmall ? 0 : '19px',
+        overflow: 'hidden',
+        width: upToSmall ? '100%' : 'fit-content',
+      }}
+    >
+      <TabButton
+        style={style}
```

Comment:
dont use inline-style here. If you need to customize TabButton, just add `className` props into that component then re-style it here. like this:
```
const StyledTabButton = styled(TabButton)`
....
`
```

Reply:
is this updated @dinhtienn? I see you resolved this but I didn't see any change

Reply:
@viet-nv Sorry It's a misunderstanding